dictionaries/software-terms/softwareTerms.txt

```diff
@@ -1192,7 +1192,11 @@ transcluded
 transclusion
 transparent
 transparentize
+transpilation
+transpile
```

Comment:
Doesn't need to be in both upper and lower case, but you could change the next one to be lower case for consistency